src/Application.php

```diff
@@ -109,67 +106,55 @@ public function __construct(
     }
 
     /**
-     * @param string|Router\Route $path
```

Comment:
Please be careful with removing comments like that, you've removed the comment but haven't update the typehint in the function declaration. In that case it's fine because as I've mentioned the `$path` is possible to be string only, so the comment will be removed anyway.